overview/makerdao.md

```diff
@@ -0,0 +1,24 @@
+{% hint style="warning" %} This documentation describes certain core aspects of MakerDAO's protocol that could change over time. Be aware that parts may be inaccurate or out of date. {% endhint %}
+
+# MakerDAO
+
+MakerDAO is a decentralized entity dedicated to bringing stability to the cryptocurrency economy. Established in 2014, MakerDAO operates as an open-source project on the Ethereum blockchain, functioning as a Decentralized Autonomous Organization (DAO). The platform utilizes the Maker protocol to generate Dai, a stablecoin backed by collateral assets approved through its governance.
+
+Governance within the MakerDAO protocol revolves around the MKR token, which provides voting rights to its holders for system changes. This approach ensures transparency, responsiveness to emergencies, community-driven development, and the ongoing growth of MakerDAO, empowering users to collectively steer the protocol's direction and maintain the stability and integrity of the ecosystem.
+
+On-Chain voting takes place in [Maker's Governance Portal](https://vote.makerdao.com/), a platform that constitutes a core component of MakerDAO's decentralized governance system. MKR holders can use their voting rights to ratify crucial protocol changes. They can also [delegate](https://manual.makerdao.com/delegation/overview) their MKR to a Delegate that aligns with their vision for the protocol.
```

Comment:
I would either edit the delegation stuff to mention AVCs or remove if it's too messy

Reply:
I think you want this section to explain:
1. What is a DAO?
2. MakerDAO governs the Maker Protocol
3. This is done through voting using MKR.